app/services/UIBuilder.scala

```diff
@@ -189,4 +191,11 @@ class UIBuilder {
     val texts: Seq[Text] = ycg.group.map(c => TextBuilder.fromPhrase(c.text))
     ConfirmationPanel(texts.head, texts.tail)
   }
+  private def fromSectionAndNoteGroup(caption: Text, ng: NoteGroup)(implicit stanzaIdToUrlMap: Map[String, String]): UIComponent = {
```

Comment:
can probably remove the implicit stanzaIdToUrlMap off these and moany other from*() fns

Reply:
as discussed, still needed due to TextBuilder